cli/src/tasks/new-plugin.ts

```diff
@@ -167,24 +167,24 @@ async function createIosPlugin(config: Config, pluginPath: string, domain: strin
 }
 
 function generatePodspec(config: Config, answers: NewPluginAnswers) {
-  return `
-  require 'json'
-
-  package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
-
-  Pod::Spec.new do |s|
-    s.name = '${fixName(answers.name)}'
-    s.version = package['version']
-    s.summary = package['description']
-    s.license = package['license']
-    s.homepage = package['repository']['url']
-    s.author = package['author']
-    s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
-    s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
-    s.ios.deployment_target  = '${config.ios.minVersion}'
-    s.dependency 'Capacitor'
-    s.swift_version = '5.0'
-  end`;
+  return `require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name = '${fixName(answers.name)}'
+  s.version = package['version']
+  s.summary = package['description']
+  s.license = package['license']
+  s.homepage = package['repository']['url']
+  s.author = package['author']
+  s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
+  s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
+  s.ios.deployment_target  = '${config.ios.minVersion}'
+  s.dependency 'Capacitor'
+  s.swift_version = '5.0'
```

Comment:
The swift version can be bumped since 5.1 is the oldest version supported by Xcode 11.
```suggestion
  s.swift_version = '5.1'
```